lib/xapian_db/query_parser.rb

```diff
@@ -35,18 +35,28 @@ def parse(expression)
 
       # Add the searchable prefixes to allow searches by field
       # (like "name:Kogler")
+      processors = [] # add processors (currently only one maximum) to scope in order to avoid
```

Comment:
I guess this fixes the segfault? If so, please add a small comment why we are collecting the processors in an array. so we don't reason about this unused array a while later ;-)